dashboard/src/features/accounts/components/AccountShow.jsx

```diff
@@ -51,7 +52,10 @@ class AccountShow extends BaseShow {
             actions={[
               <button key='show-txs' className='btn btn-link' onClick={this.props.showTransactions.bind(this, item)}>Transactions</button>,
               <button key='show-balances' className='btn btn-link' onClick={this.props.showBalances.bind(this, item)}>Balances</button>,
-              <RawJsonButton key='raw-json' item={item} />
+              <RawJsonButton key='raw-json' item={item} />,
+              <Link to={`/accounts/${item.id}/tags`} key='edit-account-tags' className='btn btn-link'>
```

Comment:
Now that we can't edit aliases, calling this "Edit" feels weird. Without consideration for implementation time, which of these do you prefer:

1. Changing this to "Edit tags"
2. Add an "edit" link in the right column of the "Tags" row.